src/jit/compiler.h

```diff
@@ -7453,7 +7468,8 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         bool compUseFCOMI;
         bool compUseCMOV;
 #ifdef _TARGET_XARCH_
-        bool compCanUseSSE2; // Allow CodeGen to use "movq XMM" instructions
+        bool compCanUseSSE2;   // Allow CodeGen to use "movq XMM" instructions
+        bool compCanUseSSE3_4; // Allow CodeGen to use SSE3, SSSE3, SSE4.1 and SSE4.2 instructions
 
```

Comment:
You could of course update the VM naming, as well -- it's only referenced about 3 times in the VM.

---
In reply to: [89898013](https://github.com/dotnet/coreclr/pull/8291#discussion_r89898013) [](ancestors = 89898013,89868211,89865129)